config/kafka-config.js

```diff
@@ -44,10 +45,29 @@ var connect = function (config) {
   })
 
   
-  _sendToKafkaConsumers(config.topics["notificationsTopic"],client, true);
-  _sendToKafkaConsumers(config.topics["languagesTopic"],client,true);
-  _sendToKafkaConsumers(config.topics["emailTopic"],client, false);
-  _sendToKafkaConsumers(config.topics["appConfigTopic"],client, true);
+  _sendToKafkaConsumers(
+    config.topics["notificationsTopic"],
+    config.host, 
+    true
+  );
+
+  _sendToKafkaConsumers(
+    config.topics["languagesTopic"],
+    config.host,
+    true
+  );
+
+  _sendToKafkaConsumers(
+    config.topics["emailTopic"],
+    config.host, 
+    false
```

Comment:
@aman-tunerlabs - Let us remove that commit setting, let it be commonly true for all.